### PR TITLE
Changed format_version default value to 4

### PR DESCRIPTION
**Pull request by jiaqizho**
### First-time contributors' checklist 

- [x] I've signed [**Contributor License Agreement**](https://cla-assistant.io/pingcap/docs) that's required for repo owners to accept my contribution.

### What is changed, added or deleted? (Required)

- changed `format_version`  default value to 4

### Which TiDB version(s) do your changes apply to? (Required)

**Tips for choosing the affected version(s):**

By default, **CHOOSE MASTER ONLY** so your changes will be applied to the next TiDB major or minor releases. If your PR involves a product feature behavior change or a compatibility change, **CHOOSE THE AFFECTED RELEASE BRANCH(ES) AND MASTER**.

For details, see [tips for choosing the affected versions](https://github.com/pingcap/docs/blob/master/CONTRIBUTING.md#guideline-for-choosing-the-affected-versions).

- [x] master (the latest development version)
- [x] v6.1 (TiDB 6.1 versions)
- [ ] v6.0 (TiDB 6.0 versions)
- [ ] v5.4 (TiDB 5.4 versions)
- [ ] v5.3 (TiDB 5.3 versions)
- [ ] v5.2 (TiDB 5.2 versions)
- [ ] v5.1 (TiDB 5.1 versions)
- [ ] v5.0 (TiDB 5.0 versions)
- [ ] v4.0 (TiDB 4.0 versions)

### What is the related PR or file link(s)?

- This PR is translated from: https://github.com/pingcap/docs-cn/pull/9507
- Other reference link(s):

### Do your changes match any of the following descriptions?

- [ ] Delete files
- [ ] Change aliases
- [ ] Need modification after applied to another branch 
- [ ] Might cause conflicts after applied to another branch

### CODE REVIEW

File: tiflash/tiflash-configuration.md [hunk 1]
```diff
@@ -77,7 +77,8 @@ delta_index_cache_size = 0
     ## * format_version = 1, the old format, deprecated.
     ## * format_version = 2, the default format for versions < v6.0.0.
     ## * format_version = 3, the default format for versions >= v6.0.0, which provides more data validation features.
```
Comment (JaySon-Huang) [inline tiflash/tiflash-configuration.md:79]:
```suggestion
    ## * format_version = 3, the default format for versions v6.0.0, which provides more data validation features.
```

File: tiflash/tiflash-configuration.md [hunk 2]
```diff
@@ -77,7 +77,8 @@ delta_index_cache_size = 0
...
-    # format_version = 3
+    ## * format_version = 4, the default format for versions >= v6.1.0, which provides PageFormat::V3. PageFormat::V3 is compatible with previous PageFormat versions
```
Comment (JaySon-Huang) [inline tiflash/tiflash-configuration.md:80]:
```suggestion
    ## * format_version = 4, the default format for versions >= v6.1.0, which provides lower write amplification and lower background task resource consumption
```

File: tiflash/tiflash-configuration.md [hunk 3]
```diff
@@ -76,8 +76,9 @@ delta_index_cache_size = 0
     ## DTFile format
     ## * format_version = 1, the old format, deprecated.
     ## * format_version = 2, the default format for versions < v6.0.0.
-    ## * format_version = 3, the default format for versions >= v6.0.0, which provides more data validation features.
-    # format_version = 3
+    ## * format_version = 3, the default format for versions v6.0.0, which provides more data validation features.
```
Comment (shichun-0415) [inline tiflash/tiflash-configuration.md:79]:
```suggestion
    ## * format_version = 3, the default format for v6.0.0, which provides more data validation features.
```

File: tiflash/tiflash-configuration.md [hunk 4]
```diff
@@ -76,8 +76,9 @@ delta_index_cache_size = 0
...
+    ## * format_version = 4, the default format for versions >= v6.1.0, which provides lower write amplification and lower background task resource consumption
```
Comment (shichun-0415) [inline tiflash/tiflash-configuration.md:80]:
```suggestion
    ## * format_version = 4, the default format for v6.1.0 and later versions, which provides lower write amplification and lower background task resource consumption
```

**Review by shichun-0415 — APPROVED** [review]
Rest LGTM

**Review by JaySon-Huang — APPROVED** [review]
LGTM

**Pull request merged**